sites/all/modules/reol_app_feeds/reol_app_feeds.install

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the reol_app_feeds module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function reol_app_feeds_install() {
+  if (module_exists('ereol_app_feeds')) {
+    // Migrate settings from ereol_app_feeds module.
+    $variable_names = [
+      'ereol_app_feeds_frontpage' => 'reol_app_feeds_frontpage',
+      'ereol_app_feeds_category' => 'reol_app_feeds_category',
+      'ereol_app_feeds_themes' => 'reol_app_feeds_themes',
+    ];
+    foreach ($variable_names as $old_name => $name) {
+      $value = variable_get($old_name);
+      if (NULL !== $value && NULL === variable_get($name)) {
+        variable_set($name, $value);
+        drupal_set_message(t('Value of variable !old_name migrated to !name.', ['!old_name' => $old_name, '!name' => $name]));
```

Comment:
Normally we use drush to enabled/disable module or update hooks. So I don't known if this would output anything (as the function uses session to store the information). So who will see the message the first logged in user?

Reply:
The message will be shown in the terminal when running `drush`. I'll remove it since nobody will see it anyways.